sig-release/members.md

```diff
@@ -0,0 +1,8 @@
+## SIG Release Membership
```

Comment:
@idvoretskyi, I think we'll need to store the list of members (by GitHub id) somewhere in order to expose the membership of GitHub teams to non org members. Given that essentially any contribution to the project comes a PR _sonewhere_ I think it's perfectly reasonable to expect people who want to declare their membership to file a PR to update the table themselves. People listed here should be more than passively invested in the goals of the SIG.